js/views/pauseScreen.js

```diff
@@ -253,8 +311,17 @@ var PauseScreen = createReactClass({
       this.props.controller.state.scrubberBar.isHovering
     );
 
+    var pauseScreenClass = ClassNames({
+      'oo-state-screen': true,
+      'oo-pause-screen': true,
+      'oo-pause-screen-hidden': this.state.hidden
+    });
+
     return (
-      <div className="oo-state-screen oo-pause-screen">
+      <div className={pauseScreenClass}
+        onMouseEnter={this.handleMouseEnter}
```

Comment:
Would this cover cases in which the pause screen is triggered via keyboard controls?

Reply:
I'll ask Steve how autohide should work with Keyboard controls.

Reply:
After talking with Steve, we'll handle keyboard controls in a separate ticket.